sdk/sdktypes/session_log_record.go

```diff
@@ -120,6 +120,15 @@ func NewCallSpecSessionLogRecord(s SessionCallSpec) SessionLogRecord {
 func (r SessionLogRecord) WithoutTimestamp() SessionLogRecord {
 	m := r.read()
 	m.T = nil
+
+	if m.CallAttemptStart != nil {
```

Comment:
This is usually a code smell, but I'm not sure what we can do here.

Reply:
i feel you